### PR TITLE
fix: cropper handle manipulation

### PR DESCRIPTION
**Pull request by rodrigobasilio2022**
### Context

Fix handle manipulation when the view direction is parallel to the line manipulator line direction.
To verify the issue, go to the cropper example

https://kitware.github.io/vtk-js/examples/ImageCroppingWidget.html

and without any rotation try to manipulate the center of the screen handle.

I´ve put an check to not allow changes if the manipulator lineDirection is parallel to the viewPlaneNormal

### Changes

- [ ] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist

- [ ] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Testing

- [ ] This change adds or fixes unit tests 
- [ ] Tested environment:
  - **vtk.js**: 
  - **OS**: 
  - **Browser**:

### CODE REVIEW

File: Sources/Widgets/Manipulators/LineManipulator/index.js [hunk 1]
```diff
@@ -19,6 +20,14 @@ export function projectDisplayToLine(
...
   vtkMath.cross(lineDirection, viewDir, normal);
   vtkMath.cross(normal, viewDir, normal);
+  // if the active camera viewPlaneNormal and line direction are parallel, no change is allowed
+  const dotProduct = Math.abs(
+    vtkMath.dot(renderer.getActiveCamera().getViewPlaneNormal(), lineDirection)
+  );
+
+  if (1 - dotProduct < EPSILON) {
+    return [];
+  }
```
Comment (finetjul) [inline Sources/Widgets/Manipulators/LineManipulator/index.js:30]: this early return could be done at the beginning of the function

Comment (rodrigobasilio2022) [reply Sources/Widgets/Manipulators/LineManipulator/index.js:30]: Done

File: Sources/Widgets/Manipulators/LineManipulator/index.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
 
 import vtkAbstractManipulator from 'vtk.js/Sources/Widgets/Manipulators/AbstractManipulator';
 
+const EPSILON = 1e-4;
```
[the remark this answers is not in the document]
Comment (rodrigobasilio2022) [reply Sources/Widgets/Manipulators/LineManipulator/index.js]: Done

**Review by finetjul — APPROVED** [review]
LGTM

**Comment by floryst** [comment]
LGTM!

**Pull request merged**